SCons/Script/Main.py

```diff
@@ -1390,7 +1390,42 @@ def _exec_main(parser, values) -> None:
 
     if isinstance(options.debug, list) and "pdb" in options.debug:
         import pdb
-        pdb.Pdb().runcall(_main, parser)
+
+        class SConsPdb(pdb.Pdb):
+            """Specialization of Pdb to help find SConscript files."""
+
+            def lookupmodule(self, filename: str) -> Optional[str]:
+                """Helper function for break/clear parsing -- SCons version.
+
+                Copy of the original, but recognizes common names for
+                SConstruct/SConscript without having to have them end in ``.py``
+
+                .. versionadded:: 4.5.0
+                """
+                if os.path.isabs(filename) and os.path.exists(filename):
+                    return filename
+                f = os.path.join(sys.path[0], filename)
+                if os.path.exists(f) and self.canonic(f) == self.mainpyfile:
+                    return f
+                root, ext = os.path.splitext(filename)
+                SCONSCRIPTS = (
```

Comment:
Can you use `KNOWN_SCONSTRUCT_NAMES` defined on line 70 instead of declaring the file names here?

Reply:
That was a disjoint addition, but no, not directly... it just has sconstruct names, not sconscript names.

Reply:
Can you push SCONSCRIPTS to the top of the file neare KNOWN_SCONSTRUCT_NAMES, so if we were to update it, it wouldn't need to be changed in more than one place?